src/renderers/dom/stack/client/ReactDOMComponent.js

```diff
@@ -953,10 +953,7 @@ ReactDOMComponent.Mixin = {
         // Do nothing for event names.
       } else if (isCustomComponent(this._tag, lastProps)) {
         if (!RESERVED_PROPS.hasOwnProperty(propKey)) {
-          DOMPropertyOperations.deleteValueForAttribute(
-            getNode(this),
-            propKey
-          );
+          DOMPropertyOperations.deleteValueForProperty(getNode(this), propKey, true);
```

Comment:
React props are now dom element properties, not attributes. The `true` is to indicate that this is a custom element node.